src/backend/people/settings.py

```diff
@@ -512,6 +512,11 @@ class Base(Configuration):
         environ_name="MAIL_PROVISIONING_API_CREDENTIALS",
         environ_prefix=None,
     )
+    DNS_PROVISIONING_TARGET_ZONE = values.Value(
+        default="collectivite.fr",
```

Comment:
Since its in common settings, please don't use this default :)

Reply:
@qbey What's a suitable alternative ? Note that the variable is only used by the plugin anyway, so unlike common code which would impact all users, this particular setting has no effect on anyone who does not use the plugin.